catapult/plugins/apps.py

```diff
@@ -18,13 +18,38 @@
 import re
 
 from catapult.api import Plugin
+from catapult.api import PreferencesItem
 from catapult.api import SearchResult
 from catapult.i18n import _
 from gi.repository import Gio
+from gi.repository import Gtk
+
+
+class AppsPluginPrefs(PreferencesItem):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.label = Gtk.Label(label=_("Ignore OnlyShowIn Attr"))
```

Comment:
The label is a bit overly technical, how about

```python
desktop = catapult.util.get_desktop_environment() or _("native")
label = _("Show non-{} apps").format(desktop)
self.label = Gtk.Label(label=label)
```

Reply:
Ok - attempting new push